ansible/roles/ansible-postgres_patroni/defaults/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+# defaults file for ansible-postgres_patroni
+#patroni .yaml config
+Postgres_cluster_name: postgresql-prod
```

Comment:
Can we use small case `p` here to keep the structure consistent with other vars?

Reply:
comments updated